src/components/NoConnectionIndicator/NoConnectionIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import { ConnectionStatus } from '@/hooks'
+import { Text } from '@/shared/components'
+import { transitions } from '@/shared/theme'
+import React from 'react'
+import { CSSTransition } from 'react-transition-group'
+import { IndicatorWrapper } from './NoConnectionIndicator.style'
+
+type NoConnectionIndicatorProps = {
+  connectionStatus: ConnectionStatus
+}
+
+const NoConnectionIndicator: React.FC<NoConnectionIndicatorProps> = ({ connectionStatus }) => {
```

Comment:
so the designs are ready:
[https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6744%3A354](https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6744%3A354)

Might be a good idea to update the PR with it